src/pyramid/view.py

```diff
@@ -237,14 +245,13 @@ def callback(context, name, ob):
             if settings.get('attr') is None:
                 settings['attr'] = wrapped.__name__
 
-        settings['_info'] = info.codeinfo  # fbo "action_method"
```

Comment:
The only benefit I see from also using this `info.codeinfo` is that it can tell us where the view is actually located which we are wrapping.

For example if you import a view function from another module, and then "decorate" it and store it in the local module, we could point the user at where their view code is located, as well as where the `view_config` decorator is being applied.

May not be worth it though, so if there are no objections to anything else in this changeset, ignore this comment.